mmdet/core/anchor/point_target.py

```diff
@@ -0,0 +1,162 @@
+import torch
+
+from ..bbox import assign_and_sample, build_assigner, PseudoSampler
+from ..utils import multi_apply
+
+
+def point_target(proposals_list,
```

Comment:
I suggest moving the target computation methods to the corresponding head. The same refactoring is also planned to be applied to other target methods.